src/Microsoft.IdentityModel.Protocols.SignedHttpRequest/SignedHttpRequestClaimTypes.cs

```diff
@@ -0,0 +1,86 @@
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+namespace Microsoft.IdentityModel.Protocols.SignedHttpRequest
+{
+    /// <summary>
+    /// Claim types used with SignedHttpRequest.
+    /// </summary>
+    public static class SignedHttpRequestClaimTypes
+    {
+        /// <summary>
+        /// https://tools.ietf.org/html/draft-ietf-oauth-signed-http-request-03#section-3
+        /// </summary>
+        public const string At = "at";
+
+        /// <summary>
+        /// https://tools.ietf.org/html/draft-ietf-oauth-signed-http-request-03#section-3
+        /// </summary>
+        public const string Ts = "ts";
+
+        /// <summary>
+        /// https://tools.ietf.org/html/draft-ietf-oauth-signed-http-request-03#section-3
+        /// </summary>
+        public const string M = "m";
+
+        /// <summary>
+        /// https://tools.ietf.org/html/draft-ietf-oauth-signed-http-request-03#section-3
+        /// </summary>
+        public const string U = "u";
+
+        /// <summary>
+        /// https://tools.ietf.org/html/draft-ietf-oauth-signed-http-request-03#section-3
+        /// </summary>
+        public const string P = "p";
+
+        /// <summary>
+        /// https://tools.ietf.org/html/draft-ietf-oauth-signed-http-request-03#section-3
+        /// </summary>
+        public const string Q = "q";
+
+        /// <summary>
+        /// https://tools.ietf.org/html/draft-ietf-oauth-signed-http-request-03#section-3
+        /// </summary>
+        public const string H = "h";
+
+        /// <summary>
+        /// https://tools.ietf.org/html/draft-ietf-oauth-signed-http-request-03#section-3
+        /// </summary>
+        public const string B = "b";
+
+        /// <summary>
+        /// The "cnf" claim might be present in SignedHttpRequest payload.
+        ///  https://tools.ietf.org/html/rfc7800#section-6.1.1
+        /// </summary>
+        public const string Cnf = "cnf";
```

Comment:
This is also in ConfirmationClaimTypes.cs

Reply:
Would you prefer keeping the member here and getting the value from ConfirmationClaimTypes or removing them from SignedHttpRequestClaimTypes?

Reply:
Addressed with https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1260/commits/67d5dad46791ea015501fdc414f65ac6afb60a1e